Cargo.lock

```diff
@@ -4646,8 +4646,8 @@ dependencies = [
 
 [[package]]
 name = "pallet-grandpa"
-version = "3.0.0"
-source = "git+https://github.com/paritytech/substrate?branch=master#1d7f6e12c651d776fc0dc1adefd007bb60f60b63"
+version = "3.1.0"
+source = "git+https://github.com/paritytech/substrate?branch=jon/migrate-grandpa-pallet-framev2#32ba95b6eac24b79de07bc0e2b64e1f5786c028c"
```

Comment:
Added this to make the substrate PR companion check go green. I hoping here that the `cargo update` that processbot does when merging will reset this. Otherwise I'll have to revert this before merging the companion